src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => (a.toString() > b.toString() ? 1 : -1),
+  ) {
+    let thisLenght = this.length;
+
+    while (thisLenght > 0) {
+      for (let i = 0; i < thisLenght - 1; i++) {
+        const current = this[i];
+        const next = this[i + 1];
+
+        if (compareFunction(this[i], this[i + 1])) {
```

Comment:
The condition in the `if` statement should explicitly check for a value greater than 0. As written, any truthy return value from `compareFunction` will enter the block, which may not always be correct because the compare function should return a number and only a positive number indicates a swap.